Dockerfile

```diff
@@ -6,9 +6,9 @@ RUN npm install
 RUN npm run build
 
 # Setup the final container that will we run
-FROM tiangolo/meinheld-gunicorn-flask:python3.7
+FROM tiangolo/meinheld-gunicorn-flask:python3.8
```

Comment:
Is there a Python 3.9 version?

Reply:
No. If there were I would have used it :(